File: my-fresh-copy/hw/codena/u1/hw06Files/Python/model/teacher.py
class Teacher:

    def __init__(self, id, name, career):
        self.id = id
        self.name = name
        self.career = career

    def to_dict(self):
        return {
            "id": self.id,
            "name": self.name,
            "career": self.career
        }

    def __str__(self):
        return (f"Teacher("
                f"id={self.id}, "
                f"name={self.name}, "
                f"career={self.career})")

File: my-fresh-copy/hw/codena/u1/hw06Files/JavaScript/model/Teacher.js
class Teacher {

    constructor(id, name, career) {
        this.id = id;
        this.name = name;
        this.career = career;
    }

    toString() {
        return `Teacher{
id=${this.id},
name=${this.name},
career=${this.career}
}`;
    }
}

export default Teacher;